cmd/spire-server/cli/run/run.go

```diff
@@ -229,6 +230,12 @@ func (cmd *Command) Run(args []string) int {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	util.SignalListener(ctx, cancel)
+	if c.LogReopener != nil {
+		go func() {
+			c.Log.Info("spawning reopener")
```

Comment:
why no `c.Log.Info("spawning reopener")` in agent code?

Reply:
Good catch. I will add it.